packages/drivers/odsp-driver/src/odspUtils.ts

```diff
@@ -149,4 +149,5 @@ export interface INewFileInfo {
     driveId: string;
     filename: string;
     filePath: string;
+    containerPackageName?: string;
```

Comment:
>containerPackageName [](start = 4, length = 20)

this should just be explicit. just take the file extension here, rather doing a bunch of mapping, default should be .fluid